src/Options/Options/ref/Microsoft.Extensions.Options.csproj

```diff
@@ -1,3 +1,4 @@
+
```

Comment:
Bet if you reverted all changes to this file, the Code Check job would pass. But, I lean toward a lot of this being an unnecessary breaking change (as @pakrym described).